code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -1533,6 +2092,26 @@
 	glass_name = "Brave Bull"
 	glass_desc = "Tequilla and coffee liquor, brought together in a mouthwatering mixture. Drink up."
 
+/datum/reagent/ethanol/caesar
+	name = "Bloody Caesar"
+	description = "Vodka, mixed with tomato juice and hot sauce. Dash with salt and pepper."
+	taste_description = "broth, protein, and spicy vodka, eh?"
+	color = "#712e2e"
+	strength = 30
+
+	glass_name = "Bloody Caesar"
+	glass_desc = "Vodka with a special tomato and hot sauce mix. Dash with salt and pepper and serve with lime."
+
+/datum/reagent/ethanol/caipirinha
+	name = "Capirinha"
+	description = "Cachaca and lime. Felicidades!"
+	taste_description = "sweet tropical alcohol and lime"
+	color = "#fafedf"
+	strength = 30
+
+	glass_name = "capirinha"
+	glass_desc = "Cachaca and lime. Felicidades!."
```

Comment:
I skimmed most of these drinks, but I noticed a few spelling issues here. Capirinha should be Caipirinha, Cachaca should be cachaça, and you have an extra period at the end of the glass_desc